networkd/network_server.ml

```diff
@@ -85,12 +87,22 @@ let set_dns_interface _ dbg ~name =
 	debug "Setting DNS interface to %s" name;
 	config := {!config with dns_interface = Some name}
 
+let compare_driver_version vs1 vs2 () =
+	try
+		let version_of vs = List.map int_of_string (String.split '.' vs) in
+		let first_non_zero lst = List.fold_left (fun a b -> if (a<>0) then a else b) 0 lst in
+		(first_non_zero (List.map2 compare (version_of vs1) (version_of vs2)) <= 0)
```

Comment:
This code looks more relaxed for comparing driver version in case version string having different count of digits. Thanks @robhoes I have updated the PR as per review comments.